ee/desktop/runner/runner.go

```diff
@@ -212,6 +212,29 @@ func (r *DesktopUsersProcessesRunner) Interrupt(interruptError error) {
 	}
 }
 
+func (r *DesktopUsersProcessesRunner) SendNotification(title, body string) error {
```

Comment:
This tied into a larger question I had about whether with this new responsibility we might want to split up the runner a bit, so the runner is only responsible for managing these processes and is separate from the menu consumer. Maybe an update for the next PR. 🙂

Reply:
I might be wrong, but I think:
1. We have one desktop process per user. It handles menu and notifications
2. the runner is responsible for managing the desktop processes
3. the root launcher spawns the runner routine

Reply:
That's how it currently works, but my point is that the runner is currently also responsible for being the consumer of menu updates from the control server (whereas the notification consumer is responsible for consuming notifications from the control server, and then uses the runner to send them). I'm wondering if it might be a better separation of responsibilities at this point if we created a new, separate menu consumer, and then the menu consumer and the notification consumer both use the runner to send updates to the desktop processes.

Reply:
I don't know what makes sense. Somewhat it's the same code, running in the same process. So it's mostly a library organization question. (With a side helping of ownership and goroutines). So it feels easy to move it around

I'm happy to go with whatever y'all think makes sense